examples/matrix-docker-ansible-deploy/vars.yml

```diff
@@ -0,0 +1,209 @@
+# SPDX-License-Identifier: 2023 - 2025 Slavi Pantaleev
+# SPDX-License-Identifier: 2023 Julian-Samuel Gebühr
+# SPDX-License-Identifier: 2025 Suguru Hirahara
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+---
+
+# Below is an example which installs a few services on the same server
+# as your Matrix services have been installed and configured with
+# matrix-docker-ansible-deploy (MDAD) playbook.
+#
+# If you are installing services on the server where neither Docker
+# nor Traefik is installed, see vars.yml on the parent directory instead.
+#
+# You should tweak this example as you see fit and enable the services that you need.
+
+########################################################################
+#                                                                      #
+# Playbook                                                             #
+#                                                                      #
+########################################################################
+
+# Put a strong secret below, generated with `pwgen -s 64 1` or in another way
+# Various other secrets will be derived from this secret automatically.
+mash_playbook_generic_secret_key: ''
+
+########################################################################
+#                                                                      #
+# /Playbook                                                            #
+#                                                                      #
+########################################################################
+
+
+########################################################################
+#                                                                      #
+# Docker                                                               #
+#                                                                      #
+########################################################################
+
+# To disable Docker SDK for Python installation (in case you'd be installing the SDK in another way),
+# remove the line below.
+devture_docker_sdk_for_python_installation_enabled: true
```

Comment:
I am not quite sure if this is indeed required or not…

Reply:
Without the SDK, Ansible's `docker.*` modules (which we use for creating container networks & pulling images & sometimes building images) can't work.

Reply:
Do you mean it can be removed because people already have it?

It doesn't really hurt to have it twice and it makes your config more portable (and similar to the ones for people who use MDAD separately).

Reply:
> It doesn't really hurt to have it twice and it makes your config more portable (and similar to the ones for people who use MDAD separately).

I understood, then I will leave it so as there will not be a race condition or conflict :wink: